docs/storage.md

```diff
@@ -307,3 +308,20 @@ config:
 ```
 
 Set the flags `--objstore.config-file` to reference to the configuration file.
+
+##  AliYun OSS Configuration
+Using AliYun OSS object storage, you should first create a bucket , with proper Storage Class & ACL. Before you access to aliyun oss, you should apply for the AccessKey . Go over [https://www.alibabacloud.com/product/oss](https://www.alibabacloud.com/product/oss) for more detail information.
+
+To configure AliYun OSS objstore need a yaml format configuration file, you should specify the parameters list below.
+
+[embedmd]:# (flags/config_oss.txt $)
+```$
+type: OSS
```

Comment:
Yes, I think OSS should be OK, because its product name is Object Storage Service, and Hadoop/Flink/... are using OSS too.

Reply:
Here is an example from Hadoop configuration.It will be better if we keep the same with other open source frameworks.

<property>
   <name>fs.oss.impl</name>
   <value>org.apache.hadoop.fs.aliyun.oss.AliyunOSSFileSystem</value>
</property>